### ISSUE TITLE
Feature Request: Modify/Backup/Restore DeviceManagement Settings

### ISSUE THREAD

**Issue by SkipToTheEndpoint** [Micke-K/IntuneManagement#91]
There are a number of additional settings available via a GET on /beta/deviceManagement, such as the GUI setting *Devices > Compliance Policies > Compliance policy settings > Mark devices with no compliance policy assigned as > Compliant/Non-compliant*, which in the API is the "secureByDefault" setting.
Other settings on this endpoint are spread about over the GUI, such as the new preview Autopilot diagnostics capture added in 2209.
Docs link: https://learn.microsoft.com/en-us/graph/api/resources/intune-deviceconfig-devicemanagementsettings?view=graph-rest-beta

Would it be possible to add something like an "Intune Tenant Settings" section? Just thinking that would be easier than trying to create GUI-related sections for them. Certain options like the secureByDefault would be incredibly useful for baselining a tenant.

The current amendable json looks like below and can be edited by a PATCH on /beta/deviceManagement with the right odata.context:
```json
{
    "settings": {
        "@odata.context": "https://graph.microsoft.com/beta/$metadata#deviceManagement/settings",
        "deviceComplianceCheckinThresholdDays": 30,
        "isScheduledActionEnabled": true,
        "secureByDefault": true,
        "enhancedJailBreak": false,
        "deviceInactivityBeforeRetirementInDay": 0,
        "derivedCredentialProvider": "notConfigured",
        "derivedCredentialUrl": null,
        "androidDeviceAdministratorEnrollmentEnabled": false,
        "ignoreDevicesForUnsupportedSettingsEnabled": false,
        "enableLogCollection": true,
        "enableAutopilotDiagnostics": true,
        "enableEnhancedTroubleshootingExperience": true,
        "enableDeviceGroupMembershipReport": true
    }
}

**Comment by Micke-K** [comment]
Hello!

This should be possible. I'll see if I can have  a look at it at some point.

Not sure how this would be implemented in the best way. I can't do Export/Import like all the other values. Might add something to the Views menu for this. Not sure if that would be the best solution either but might be a start

Cheers!

**Comment by Micke-K** [comment]
Hello!

This was included in 3.9 in the Intune Info view. Is that ok? 
So it will not work with Bulk Export/Import.

Cheers!

**Comment by Micke-K** [comment]
Closing the issue since it was partially included in 3.9.

Cheers!
